packages/vscode-extension/src/webview/components/DeviceSettingsDropdown.tsx

```diff
@@ -112,6 +113,26 @@ function DeviceSettingsDropdown({ children, disabled }: DeviceSettingsDropdownPr
             <span className="codicon codicon-location" />
             Set Device Location
           </DropdownMenu.Item>
+          <Label>Device Frame</Label>
+          <form>
+            <div style={{ display: "flex", alignItems: "center" }}>
+              <label className="Label" htmlFor="device-frame" style={{ paddingRight: 15 }}>
```

Comment:
@jakub-gonet Is there a sense to use this component? This component currently is a p tag what does not play well with accessibility, for instance cannot toggle clicking the label. The two ideas I have is to remove the `Label` and just use the className since this component does not do anything, zero custom logic, or to fix the label component tag, change it to label and allow passing label props to it. What do you think make more sense?

Reply:
`<Label />` is shared which makes it more difficult. We can try to change it to use `label` and pass optional `for` prop. If that breaks some of the uses then I think we should continue to use `<Label />` as is for consistency, even if broken.

We can always refactor later instead of overcomplicating this PR.